src/status.cpp

```diff
@@ -47,6 +47,9 @@ const std::map<const StatusCode, const std::string> Status::statusMessageMap = {
     {StatusCode::INVALID_NIREQ, "Nireq parameter too high"},
     {StatusCode::REQUESTED_DYNAMIC_PARAMETERS_ON_SUBSCRIBED_MODEL, "Requested dynamic parameters but model is subscribed to pipeline"},
     {StatusCode::PIPELINE_STREAM_ID_NOT_READY_YET, "Node is not ready for execution"},
+    {StatusCode::REQUESTED_DYNAMIC_PARAMETERS_ON_STATEFUL_MODEL, "Dynamic shape and dynamic batch size not supported for stateful models"},
+    {StatusCode::REQUESTED_SUBSCRIBED_MODEL_AND_STATEFUL_MODEL, "Stateful model cannot be subscribed to pipeline"},
```

Comment:
```suggestion
    {StatusCode::REQUESTED_SUBSCRIBED_MODEL_AND_STATEFUL_MODEL, "Stateful model cannot be used in pipeline"},
```

Reply:
I think that we should have the comment changed as well in line 48:
"Requested dynamic parameters but model is subscribed to pipeline"
to
"Requested dynamic parameters but model is used in pipeline"
As those would be easier to understand to OVMS admin.